src/components/card-with-button/СardWithButton.js

```diff
@@ -0,0 +1,58 @@
+import { Box, Button } from '@mui/material'
+import { useContext, useMemo } from 'react'
+import { useTranslation } from 'react-i18next'
+import { ModalContext } from '~/context/modal-context'
+import CardWithImage from '~/components/card-with-image/CardWithImage'
+import SignupDialog from '~/containers/guest-home-page/signup-dialog/SignupDialog'
+
+export const styles = {

```

Comment:
We need to export only the component from the component file. If you have something to reuse in multiple places, move it to constants, or utils if it is a method.

Reply:
fixed